site/content/testimonials/maria-karamitsou.md

```diff
@@ -8,8 +8,6 @@ image: /images/testimonials/karamitsou_th.jpg
 leadquote: "I found lecture notes, handouts and slides from presentations, and some problem sets. It helped me a lot."
 ---
 
-# Maria Karamitsou
-
 A fifth-year civil engineering student at Aristotle University of Thessaloniki
```

Comment:
Likely outside the scope of this PR but the wavelets link below goes to the legacy site instead of this one

Reply:
I'm assuming there's a lot of content cleanup that we'll need to do. Agree that it's not in scope